include/widgets-options-filters.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Class PLL_Widgets_Filters
+ *
+ * Add new options to {@see https://developer.wordpress.org/reference/classes/wp_widget/ WP_Widget} and saves them.
+ */
+class PLL_Widgets_Options_Filters {
```

Comment:
For consistency with other class names, I suggest to use `PLL_Filters_Widgets_Options`.